packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/flex.dart

```diff
@@ -22,8 +22,8 @@ import 'free_space.dart';
 import 'overflow_indicator_painter.dart';
 import 'utils.dart';
 
-const widthIndicatorColor = mainUiColor;
-const heightIndicatorColor = Color(0xFF27AAE1);
+const widthIndicatorColor = ThemedColor(Color(0xFF000099), Color(0xFF9EBEF9));
```

Comment:
I grab this color from mainUiColor

Reply:
You should be able to pass mainUiColor in as well.